Composer/packages/extension-client/src/hooks/useMenuConfig.ts

```diff
@@ -29,6 +33,21 @@ export function useMenuConfig(): { menuSchema: MenuUISchema; forceDisabledAction
       }
     });
 
+    // Add sdk actions missing in uischema
+    Object.entries(sdkDefinitions).forEach(([$kind, action]) => {
+      if (
+        $kind.startsWith('Microsoft') &&
+        $kind !== SDKKinds.AdaptiveDialog &&
+        !implementedMenuSchema[$kind] &&
+        actionImplementsDialog(action)
+      ) {
+        implementedMenuSchema[$kind] = {
+          label: getFallbackLabel($kind),
+          submenu: ['Other'],
```

Comment:
Should we localize this? If so, we need to add the format message package.

Reply:
@yeze322 would know more here but I think the menu is currently not localized or localized elsewhere.